src/commissaire/constants.py

```diff
@@ -40,6 +40,13 @@
     'type': NETWORK_TYPE_DEFAULT
 }
 
+#: Container Manager type for OpenShift
+CONTAINER_MANAGER_OPENSHIFT = 'openshift'
+#: Container Manager type to use when a default is required
+CONTAINER_MANAGER_DEFAULT = CONTAINER_MANAGER_OPENSHIFT
+#: All Container Manager types
+CONTAINER_MANAGER_TYPES = [CONTAINER_MANAGER_OPENSHIFT]
+
```

Comment:
I expected to see all the `CLUSTER_TYPE_*` constants removed.  We don't need them anymore, right?